### DISCUSSION TITLE
Track artist instead of album artist?

### DISCUSSION THREAD

**Discussion by SightSpirit** [Radiicall/jellyfin-rpc#73] [Ideas]
I listen to a lot of compilation albums, so jellyfin-rpc's use of the album artist in the by-line is often not representative of what I'm listening to. Is there any way to set the program to pull from the artist field instead? If not, would it be difficult to implement?

Thanks!

**Comment by justradical** [comment]
This is being worked on!

**Comment by justradical** [comment]
Made issue from this post #74
